src/aat/resources/cucumber/questions.feature

```diff
@@ -31,3 +31,48 @@ Feature: Questions feature
     When the get request is sent to retrieve all questions
     Then the response code is 200
     And the response contains 2 questions
+
+  Scenario: Edit the question body
+    Given a standard online hearing is created
+    And a standard question
+    When the post request is sent to create the question
+    Then the response code is 200
+    Given the question body is edited to ' "some new question text?" '
+    When the put request to update the question is sent
+    Then the response code is 200
+    When the get request is sent to retrieve the submitted question
+    Then the response code is 200
+    And the question body is ' "some new question text?" '
+
+  Scenario: Edit the question header
+    Given a standard online hearing is created
+    And a standard question
+    When the post request is sent to create the question
+    Then the response code is 200
+    Given the question header is edited to ' "some new header text?" '
+    When the put request to update the question is sent
+    Then the response code is 200
+    When the get request is sent to retrieve the submitted question
+    Then the response code is 200
+    And the question header is ' "some new header text?" '
+
+  Scenario: Edit the question state to submitted
+    Given a standard online hearing is created
+    And a standard question
+    When the post request is sent to create the question
+    Then the response code is 200
+    Given the question state is edited to ' "SUBMITTED" '
+    When the put request to update the question is sent
+    Then the response code is 200
+    When the get request is sent to retrieve the submitted question
+    Then the response code is 200
+    And the question state name is SUBMITTED
```

Comment:
You're correct in having a submitted state, which isn't in the HLD.  Let's talk about this Monday.